README.md

```diff
@@ -157,6 +160,7 @@ cd multipy/runtime
 mkdir build
 cd build
 # use cmake -DABI_EQUALS_1=ON .. instead if you want ABI=1
+# if you built PyTorch from source, pass -DPYTORCH_ROOT=path/to/pytorch
```

Comment:
This line isn't needed with recent changes. Also does this work given we always set it in cmake?